src/api/eventTypes.js

```diff
@@ -154,6 +154,18 @@ export type UpdateMessageFlagsEvent = $ReadOnly<{|
   flag: empty, // TODO fill in
   all: boolean,
   messages: $ReadOnlyArray<number>,
+
+  message_info_dict?: {|
```

Comment:
I'm not sure I see what the idea behind the `TODO` would be — other examples of it I see are for marking things as required or dropping them, but this field will need to be optional for the foreseeable future, since it's not used for `op: add`.

I've added a comment stating what version it was added in, but I'm not convinced that's generally useful, given that we don't seem to have a habit of doing that elsewhere in this file.

Reply:
Hmm I see, right, it remains optional even in the new API.

Yeah, I don't think there's a need for a comment when there isn't something actionable for us to eventually do with it.